regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -33,13 +40,30 @@ var PreambleView = ChildView.extend({
     this.$read.show();
   },
 
-  handleWrite: function(e) {
+  handleParagraphActive: function(id) {
+    // update current Section ID as user scrolls
+    this.currentSectionId = id;
+  },
+
+  handleWrite: function() {
```

Comment:
This view now has two write listeners, `handleWrite` and `handleWriteLink`. To clarify when each handler gets called, how about renaming this one to something like `handleWriteEvent`?